src/Amido.Stacks.SQS/Consumer/EventConsumer.cs

```diff
@@ -0,0 +1,38 @@
+using Amazon.SQS;
+using Amazon.SQS.Model;
+using Amido.Stacks.Configuration;
+using Microsoft.Extensions.Options;
+
+namespace Amido.Stacks.SQS.Consumer
+{
+    /// <summary>
+    /// Class implementing the ability to consume to an event on AWS SQS
+    /// </summary>
+    public class EventConsumer : IEventConsumer
+    {
+        private readonly IAmazonSQS _queueClient;
+        private readonly IOptions<AwsSqsConfiguration> _configuration;
+        private readonly ISecretResolver<string> _secretResolver;
+
+        public EventConsumer(
+            IOptions<AwsSqsConfiguration> configuration,
+            ISecretResolver<string> secretResolver,
+            IAmazonSQS queueClient)
+        {
+            _configuration = configuration ?? throw new ArgumentNullException(nameof(configuration));
```

Comment:
We generally use `this` instead of the underscore approach.

Reply:
Thanks for letting me know. Will try to remember going forwards :)